experimental/FTheoryTools/src/LiteratureModels/constructors.jl

```diff
@@ -259,7 +259,7 @@ function _construct_literature_model_over_concrete_base(model_dict::Dict{String,
 
   # Find divisor classes of the internal model sections
   auxiliary_base_grading = matrix(ZZ, transpose(hcat([[eval_poly(weight, ZZ) for weight in vec] for vec in model_dict["model_data"]["auxiliary_base_grading"]]...)))
-  auxiliary_base_grading = vcat([[Int(k) for k in auxiliary_base_grading[i,:]] for i in 1:nrows(auxiliary_base_grading)]...)
+  auxiliary_base_grading = vcat([[Int(k) for k in auxiliary_base_grading[i:i,:]] for i in 1:nrows(auxiliary_base_grading)]...)
```

Comment:
Yes. It will just be a `Vector` (see the Linear algebra roadmap), so that the dimension of the indices now correspond to the dimension to the output (as for julia arrays).

1. Nowadays we have more support for `Matrix * Vector` and `solve(::Matrix, ::Vector)` and things like that, which we did not have 5 years ago. So back then we avoided `Vector` at all costs.
2. At the moment, it is cumbersome to get rows/columns as `Vector`. With the "new" indexing, it is easy: Just do `[i, :]`. To get it as a matrix, just do `[[i], :]` or `[i:i, :]`.